src/Routing/Router.php

```diff
@@ -159,6 +159,13 @@ class Router
      */
     protected static $_defaultExtensions = [];
 
+    /**
+     * Cache of parsed short strings
+     *
+     * @var array
+     */
+    protected static $_shortStrings = [];
```

Comment:
Do we need a way to clear those? For test runs etc? Or is this sufficient as is?

Reply:
`Router::reload()` resets all static properties.